src/Logging/Logging/ScopedLogService.cs

```diff
@@ -38,46 +38,69 @@ public bool IsEnabledFor(LogLevel logLevel)
         public virtual void DisableLogging() => 
             IsEnabled = false;
 
+
         public void Log<TState>(
             DateTimeOffset timestamp,
             LogLevel logLevel,
             TState state,
-            Exception ex,
-            Func<TState, Exception, string> formatter)
+            string methodName = "")
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5762132)